.github/workflows/ci.yaml

```diff
@@ -36,6 +34,11 @@ jobs:
           composer-${{ hashFiles('composer.json') }}
           composer-
     - run: make install
-    - run: make cs-check
-    - run: make phpstan
-    - run: make test
+    - name: Code style
+      if: ${{ matrix.php == '8.0' && matrix.deps == '--prefer-stable' }}
+      run: make cs-check
+    - name: Static analysis
+      if: ${{ matrix.php == '8.0' && matrix.deps == '--prefer-stable' }}
```

Comment:
Cs fixer and phpstan can now also be executed on php 8.1 https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/tag/v3.4.0

Reply:
Yeah, this is just to prevent every matrix combination to do basically the same thing. Doesn't really matter if it's done on 8.0 or 8.1